src/GeositeFramework/js/Export.js

```diff
@@ -54,63 +68,79 @@ require(['use!Geosite'],
                 maskopacity: 40,
                 openjs: function () {
                     $('#export-button').on('click', function() {
-                        var previewDeferred = $.Deferred(),
-                            postPrintAction = function() {},
-                            mapNode = _.head($("#map-0").detach());
+                        var mapNode = _.head($("#map-0").detach());
 
                         $('.print-sandbox-header h1').text($("#export-title").val());
                         $("#export-print-preview-map").append(mapNode);
 
                         mapReadyDeferred.then(function () {
                             $("#export-print-preview-map").detach().appendTo($("#print-map-container"));
 
-                            // expand legend container if minimized
-                            if ($("#legend-container-0").hasClass("minimized")) {
-                                _.head($(".legend-close")).click();
-                                postPrintAction = function() {
-                                    _.head($(".legend-close")).click();
-                                };
-                            }
-
-                            if ($("[name='export-include-legend']").is(":checked")) {
-                                // show & expand all legend items
-                                $("#legend-container-0").css({ visibility: "visible" });
-                                _.each(
-                                    $(".item.expand>.expand-legend"),
-                                    function (el) {
-                                        el.click();
-                                    });
-                                $(".item.extra.collapse").hide();
-
-                                // wrap all items in a div to style separately from header
-                            } else {
-                                // if the style rule is changed via jquery, that state seems to
-                                // "stick", regardless of what's in the stylesheet
-                                $("#legend-container-0").css({ visibility: "hidden" });
-                            }
-                            var pageOrientation = $("[name='export-orientation']").filter(function () { return this.checked; }).map(function () { return this.value; }).first()[0];
-                            var orientDeferred = $.Deferred();
 
+                            var pageOrientation = _.head($("[name='export-orientation']")
+                                .filter(function () { return this.checked; })
+                                .map(function () { return this.value; })
+                                .first());
                             if (pageOrientation === "Landscape") {
```

Comment:
These two blocks for adding the orientation css link are nearly identical.  Consider calling it out into a function that generates the string change, or just templates in the variable:

```js
$('<link>', {
    rel: 'stylesheet',
    href: 'css/print-' + pageOrientation + '.css',
    'class': '.print-orientation-css',
 }).appendTo('head');
```